frontend/src/pages/chat/components/bot-message/bot-message.tsx

```diff
@@ -0,0 +1,21 @@
+import { Icon } from '#libs/components/components.js';
+import { type InputMessage } from '#libs/types/types.js';
+
+import styles from './styles.module.scss';
+
+const BotMessage: React.FC<InputMessage> = ({ messages }) => {
```

Comment:
```suggestion
const BotMessage: React.FC<Properties> = ({ messages }) => {
```

Reply:
And create it in this file

Reply:
Done, this approach is well? 

<img width="540" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/90299049/eb2f5fe6-e2e0-42d7-bbf8-f8b1fa0aaa14">